### PR TITLE
feature(component): Showcase Performer Profile

### PR DESCRIPTION
**Pull request by gepifanio**
Ticket: [#235](https://github.com/unimelb/ffam-prototype/issues/235)

- [x] add tests

[Showcase perfomer Profile](https://deploy-preview-991--pattern-lib-unimelb.netlify.com/?selectedKind=Cards%2FProfiles&selectedStory=Showcase%20Performer%20Profile&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 2e9b03a347c2f83782120061923bf4541b62a3ed

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5c9b0076b5430800075a2f94

File: components/cards/_card-showcase-profile.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  .card__icon {
+    display: initial;
+    margin-left: 4px;
+  }
+
+  .card__bio {
+    padding: 12px 16px 12px 16px;
+    margin: 0;
+    max-width: 100%;
+    background-color: var(--col-navy-light);
```
Comment (TimMansell) [inline components/cards/_card-showcase-profile.css]: missing rgb( var(--col-navy-light))

File: components/cards/_card-showcase-profile.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    @media (--bp-max-mobile) {
```
Comment (TimMansell) [inline components/cards/_card-showcase-profile.css]: CSS should be written in mobile first.  So mobile styles should be default then use media queries to modify desktop styles.

File: components/cards/_card-showcase-profile.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    background: rgb(9, 65, 131);
+    margin: .5rem 0 .5rem 0;
+
+    @media (--bp-max-mobile) {
+      max-width: 6%;
+    }
+  }
+
+  .card__button {
+    max-width: 81px;
```
Comment (TimMansell) [inline components/cards/_card-showcase-profile.css]: Ideally, we should try and avoid magic numbers where possible.

**Comment by TimMansell** [comment]
Also, this component is similar to other cards so we can use most of the same markup and apply styling to differ.

I have refactored some of the code which I can show you.

File: components/_grid.css [hunk 4]
```diff
@@ -38,36 +47,115 @@
   padding-right: calc(var(--vr) / 2);
   padding-left: calc(var(--vr) / 2);
 
+  &--no-padding {
```
Comment (TimMansell) [inline components/_grid.css]: @gepifanio, FYI, I modified the code so the no-padding class can have more use cases. `cell--no-padding` to me means that it should always have no padding.  `cell--no-padding-mobile` etc removes padding for that specific breakpoint.

**Pull request merged**